### PR TITLE
Devl/new functionality

### PR DESCRIPTION
**Pull request by kelvin-onlab**

### CODE REVIEW

File: TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py [hunk 1]
```diff
@@ -10,6 +10,7 @@ def hostIntent( main,
                 name,
                 host1,
                 host2,
+                onosNode=0,
```
Comment (jhall11) [inline TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py]: You may want to typecast this to an int just incase it is passed straight from a params file

Comment (kelvin-onlab) [inline TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py]: Oh ok

File: TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py [hunk 2]
```diff
@@ -543,6 +550,7 @@ def singleToMultiIntent( main,
                                             tcpSrc="",
                                             tcpDst="" ) )
 
+    time.sleep( 10 )
```
Comment (jhall11) [inline TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py]: can you document what this is for?

Comment (kelvin-onlab) [inline TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py]: Okay~ I think I was waiting for the flows to go through out the instances.

File: TestON/tests/PingallExample/PingallExample.params [hunk 3]
```diff
@@ -1,12 +0,0 @@
-<PARAMS>
```
Comment (jhall11) [inline TestON/tests/PingallExample/PingallExample.params]: Did you mean to delete this example test?

Comment (kelvin-onlab) [inline TestON/tests/PingallExample/PingallExample.params]: Oh noes, I think it was my mistake, I think the one that got merge in some
commit was kaouther's pingallExample test so I was trying not to include it
in the commit I think I delete it in this commit by mistake.

**Pull request closed without merging**